src/plugins/theme/maxWidth.js

```diff
@@ -31,11 +29,6 @@ module.exports = function () {
     1000: '100rem',
     1200: '120rem',
     1500: '150rem',
-    full: '100%',
-    min: 'min-content',
-    max: 'max-content',
-    fit: 'fit-content',
-    prose: '65ch',
```

Comment:
Not actually removing these, we'll get these same options from the extend change